client/web/src/repo/blob/own/HistoryAndOwnBar.tsx

```diff
@@ -66,6 +66,9 @@ export const HistoryAndOwnBar: React.FunctionComponent<{
 
     const history = data?.node?.commit?.ancestors?.nodes?.[0]
     const ownership = data.node.commit?.blob?.ownership
+    const contributors = data.node.commit?.blob?.contributors
+    // +1 contributor (singular), but +3 contributors (plural)
+    const contributorText = contributors?.totalCount === 1 ? 'contributor' : 'contributors'
```

Comment:
with some
```js
import { pluralize } from '@sourcegraph/common'
```
we can do
```suggestion
    const contributorText = pluralize('contributor', contributors?.totalCount ?? 0)
```